src/runner/bootstrapper.ts

```diff
@@ -186,6 +188,28 @@ export default class Bootstrapper {
         return compiler.getTests();
     }
 
+    private _assertGlobalHooks () : void {
+        if (this.hooks?.fixture?.before)
+            assertType(is.function, 'globalBefore', 'The fixture.globalBefore hook', this.hooks.fixture.before);
+
+        if (this.hooks?.fixture?.after)
+            assertType(is.function, 'globalAfter', 'The fixture.globalAfter hook', this.hooks.fixture.after);
+    }
+
+    private _setGlobalHooksToTest (tests: Test[]): Test[] {
+        this._assertGlobalHooks();
+
+        const fixtureBefore = this.hooks?.fixture?.before || null;
+        const fixtureAfter = this.hooks?.fixture?.after || null;
+
+        return tests.map(item => {
+            item.fixture.globalBeforeFn = fixtureBefore;
+            item.fixture.globalAfterFn = fixtureAfter;
+
+            return item;
```

Comment:
multiple tests can have the same fixture. This means that we set the `globalBeforeFn/globalAfterFn` multiple times for the specific fixture. It can be misleading

Reply:
I would prefer not to return the array of tests from the method with the `set` name. It seems that we do not need return anything from this method.

Reply:
1. I did it for the next steps because we also need to assign 'global before/after' and 'request hook' for each test.
2. I agree with you, that we don't need to return smth. I'll fix it